core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java

```diff
@@ -25,8 +25,10 @@
 
 import org.apache.sling.api.SlingHttpServletRequest;
 import org.apache.sling.engine.EngineConstants;
-import org.osgi.framework.Constants;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
```

Comment:
@DeepaliAgrwl : why do we have this java class ?

Reply:
removed this file